### PR TITLE
Feat/page change progress bar

### PR DESCRIPTION
**Pull request by Lamperoyge**
## :pushpin: References

- **Wonder issue:**
- **Related pull-requests:**

## :blue_book: Description

Add animation on page changes

## :movie_camera: Screenshot or Video

https://www.loom.com/share/e84c5c0a233b49a7baa4037fd7e1ecf6

## :cake: Checklist:

- [x] I self-reviewed my changes
- [x] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [x] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-app/components/NavigationProgress/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  top: 0%;
+  height: 3px;
+  background-color: transparent;
+`;
+
+export const ProgressBarInner = styled.div`
+  height: 100%;
+  border-top-right-radius: 30px;
+  border-bottom-right-radius: 30px;
+  background: linear-gradient(270deg, #ccbbff -5.62%, #7427ff 45.92%, #00baff 103.12%);
```
Comment (juniusfree) [inline wondrous-app/components/NavigationProgress/styles.tsx:16]: We must use the color palette here.

File: wondrous-app/components/NavigationProgress/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+export const ProgressBarWrapper = styled.div`
```
Comment (juniusfree) [inline wondrous-app/components/NavigationProgress/styles.tsx:3]: May we use MUI Grid here?

**Review by juniusfree — COMMENTED** [review]
Nice work. I have a couple of comments.

File: wondrous-app/components/NavigationProgress/styles.tsx (hunk 1 above)
Comment (Lamperoyge) [reply wondrous-app/components/NavigationProgress/styles.tsx]: yep, done

File: wondrous-app/components/NavigationProgress/styles.tsx (hunk 2 above)
Comment (Lamperoyge) [reply wondrous-app/components/NavigationProgress/styles.tsx:4]: We could I guess but why? it's a very basic div

**Comment by terryli0095** [comment]
niceee

File: wondrous-app/components/NavigationProgress/styles.tsx (hunk 2 above)
Comment (juniusfree) [reply wondrous-app/components/NavigationProgress/styles.tsx:4]: It's up to you but I find that wrapper components like this becomes more obvious to the reader and it's reusable.
https://mui.com/system/getting-started/usage/#why-use-mui-system

Though there's some performance tradeoff
https://mui.com/system/getting-started/usage/#when-to-use-mui-system

**Pull request merged**